tools/system_libs.py

```diff
@@ -1416,17 +1416,32 @@ def can_build(self):
 class libwasmfs(MTLibrary, DebugLibrary, AsanInstrumentedLibrary):
   name = 'libwasmfs'
 
-  cflags = ['-fno-exceptions', '-std=c++17']
-
   def get_files(self):
-    return files_in_path(
+    wasmfs_files = files_in_path(
         path='system/lib/wasmfs',
         filenames=['syscalls.cpp', 'file_table.cpp', 'file.cpp', 'wasmfs.cpp',
                    'streams.cpp', 'memory_file.cpp', 'memory_file_backend.cpp',
                    'js_file_backend.cpp', 'proxied_file_backend.cpp',
                    'fetch_backend.cpp',
                    'js_api.cpp'])
 
+    # The JS filesystem implements e.g. utime() directly in JS, while WasmFS
+    # wants to use the normal musl code that does a syscall, and WasmFS
+    # implements the syscall.
+    libc_files = files_in_path(
+        path='system/lib/libc/musl/src/time',
+        filenames=['utime.c'])
```

Comment:
Any reason we can't just switch to including this file across the board?  (I can take a look as a followup if you like).